<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigModulos</class>
 <widget class="QDialog" name="ConfigModulos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuração das falhas do sistema</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Selecione a falha a ser detecteada, o tipo do módulo detecção e em seguida inclua os arquivos que deverão ser carregados para configurá-lo. Após isto, clique no botão adicionar módulo.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Falha</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="falhas">
       <property name="toolTip">
        <string>Local da falha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tipos">
       <property name="toolTip">
        <string>Local da falha</string>
       </property>
       <item>
        <property name="text">
         <string>RNA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fuzzy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Estatístico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Personalizado</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ordem"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Tipo de módulo</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_ordem">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ordem</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Lista de arquivos de configuração</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="adicionar_arq">
       <property name="toolTip">
        <string>Adicionar arquivo</string>
       </property>
       <property name="icon">
        <iconset resource="../imgs/imgs.qrc">
         <normaloff>:/imgs/abrir.png</normaloff>:/imgs/abrir.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remover_arq">
       <property name="toolTip">
        <string>Remover arquivo</string>
       </property>
       <property name="icon">
        <iconset resource="../imgs/imgs.qrc">
         <normaloff>:/imgs/cancelar.png</normaloff>:/imgs/cancelar.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="subir_arq">
       <property name="toolTip">
        <string>Subir arquivo na lista</string>
       </property>
       <property name="icon">
        <iconset resource="../imgs/imgs.qrc">
         <normaloff>:/imgs/mover_para_cima.png</normaloff>:/imgs/mover_para_cima.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="descer_arq">
       <property name="toolTip">
        <string>Descer arquivo na lista</string>
       </property>
       <property name="icon">
        <iconset resource="../imgs/imgs.qrc">
         <normaloff>:/imgs/mover_para_baixo.png</normaloff>:/imgs/mover_para_baixo.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QListWidget" name="lista_arquivos">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Lista de arquivos a serem carregados</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lista de módulos cadastrados</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>153</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="adicionar">
         <property name="toolTip">
          <string>Adicionar módulo</string>
         </property>
         <property name="icon">
          <iconset resource="../imgs/imgs.qrc">
           <normaloff>:/imgs/adicionar.png</normaloff>:/imgs/adicionar.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="remover">
         <property name="toolTip">
          <string>Remover módulo</string>
         </property>
         <property name="icon">
          <iconset resource="../imgs/imgs.qrc">
           <normaloff>:/imgs/remover.png</normaloff>:/imgs/remover.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QTableWidget" name="modulos_cad">
     <property name="toolTip">
      <string>Módulos cadastrados</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Falha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Qtde. Arquivos</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>153</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>153</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="salvar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Salvar alterações</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fechar">
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>adicionar</tabstop>
  <tabstop>remover</tabstop>
  <tabstop>modulos_cad</tabstop>
  <tabstop>fechar</tabstop>
 </tabstops>
 <resources>
  <include location="../imgs/imgs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fechar</sender>
   <signal>clicked()</signal>
   <receiver>ConfigModulos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
